main.css

```diff
@@ -9,8 +9,8 @@ body {
   background-color: #fff;
 }
 
-/* header section */
-.header-section {
+/* toolbar section */
+.toolbar-section {
   display: flex;
   flex-direction: row;
   column-gap: 17em;
```

Comment:
When I check your webpage it shows me some missing parts. You can compare it with this image.

| Original      | Yours |
  | ----------- | ----------- |
|<img width="321" alt="Screen Shot 2021-11-24 at 1 13 17 PM" src="https://user-images.githubusercontent.com/69011963/143199879-d4af11b6-fc9d-40a7-8a7a-e4a6da402650.png">| ![Screen Shot 2021-11-24 at 12 51 52 PM](https://user-images.githubusercontent.com/69011963/143199778-4708fab9-698f-4545-8414-056774eebec3.png)|

- [ ] Your hamburger button is should appear on the right and has a nav tag. Kindly fix this issue.